doc/source/getting_started/faqs.rst

```diff
@@ -237,7 +237,7 @@ learn how to use PyFluent:
 
   .. code:: python
 
-    import :(%py-exec "workflow.TaskObject['Describe Geometry and Flow'].Arguments.setState({r'AddEnclosure': r'No',r'CloseCaps': r'Yes',r'FlowType': r'Internal flow through the object',})")
+    %py-exec "workflow.TaskObject['Describe Geometry and Flow'].Arguments.setState({r'AddEnclosure': r'No',r'CloseCaps': r'Yes',r'FlowType': r'Internal flow through the object',})"
```

Comment:
I believe that removing `import` is right, but the Scheme command extracted would still have the parenthesis:
```suggestion
    (%py-exec "workflow.TaskObject['Describe Geometry and Flow'].Arguments.setState({r'AddEnclosure': r'No',r'CloseCaps': r'Yes',r'FlowType': r'Internal flow through the object',})")
```